frontend/src/apis/loginAPI.js

```diff
@@ -64,14 +64,28 @@ const loginAPI = {
     return apiPost(CONFIG.LOGIN_API_URL + 'createUser', payload)
   },
   sendEmail: async (email, email_text = 7, email_verification_code_type = 'resetPassword') => {
-    //console.log('api email', email)
     const payload = {
       email,
       email_text,
       email_verification_code_type,
     }
     return apiPost(CONFIG.LOGIN_API_URL + 'sendEmail', payload)
   },
+  loginViaEmailVerificationCode: async (optin) => {
+    return apiGet(
+      CONFIG.LOGIN_API_URL + 'loginViaEmailVerificationCode?emailVerificationCode=' + optin,
+    )
+  },
+  changePassword: async (session_id, email, password) => {
+    const payload = {
+      session_id,
+      email,
```

Comment:
We do not have the email available so far.

Reply:
Not a Solution to this problem, but consider retrieving missing info via: https://github.com/gradido/gradido/blob/master/docu/login_server.api.md#retrieve-user-data